UefiPayloadPkg/GraphicsOutputDxe/GraphicsOutput.c

```diff
@@ -409,6 +409,10 @@ GraphicsOutputDriverBindingStart (
               if (Resources->AddrRangeMin == GraphicsInfo->FrameBufferBase) {
                 FrameBufferBase = Resources->AddrRangeMin;
                 break;
+              } else if (Resources->AddrRangeMin <= GraphicsInfo->FrameBufferBase

```

Comment:
We may apply the suggestion from the other PR to reduce the number of IF branches

Reply:
Fixed it myself.